importers/vcs/git/git_dao.py

```diff
@@ -400,7 +400,7 @@ def insert_file_renamed(self, repo_id, current_file_id, previous_file_id, file_m
         self._cnx.commit()
         cursor.close()
 
-    def insert_file(self, repo_id, name, ext):
+    def insert_file(self, repo_id, name, ext=None):
```

Comment:
I want to see "extract_dependency_relations" API use independently too. In that case its needed.